### PR TITLE
[IMP] sale_order_event_attendee: search event order by line ticket

### PR DESCRIPTION
**Pull request by leiremartinezdesantos**

### CODE REVIEW

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

File: sale_order_event_attendee/models/event.py [hunk 1]
```diff
@@ -34,16 +34,13 @@ def action_confirm(self):
             order = record.sale_order_id
             if select_ticket:
                 if not order:
-                    order_obj = self.env['sale.order']
-                    order = order_obj.search([
-                        ('partner_id', '=', record.partner_id.id),
-                        ('state', 'in', ('draft', 'sent'))
-                    ], order='date_order desc', limit=1)
+                    line, order = record.get_event_order(select_ticket)
```
Comment (anajuaristi) [inline sale_order_event_attendee/models/event.py]: Hemen
```python
line, order = record.get_event_order(select_ticket)

if not line --> 
    if not order --> 
        order = self.env['sale.order'].sudo().create({'partner_id': self.partner_id.id})
    line = 
        self.env['sale.order.line'].sudo().create(
                            {
                                'product_id': record.event_ticket_id.product_id.id,
                                'event_id': record.event_id.id,
                                'event_ticket_id': record.event_ticket_id.id,
                                'name': record.event_ticket_id.name,
                                'order_id': order.id,
                                'product_uom':
                                    record.event_ticket_id.product_id.uom_id.id})

record.write({'sale_order_line_id': line.id,
              'sale_order_id': order.id})


```
Listo... ez dago  programazio gehiagorik action_confirm bukaera arte

File: sale_order_event_attendee/models/event.py [hunk 2]
```diff
@@ -34,16 +34,13 @@ def action_confirm(self):
             order = record.sale_order_id
             if select_ticket:
```
Comment (anajuaristi) [inline sale_order_event_attendee/models/event.py:35]: if not order, kendu... zuzenean exekutatu record.get_event_order(select ticket)

**Review by anajuaristi — CHANGES_REQUESTED** [review]

**Review by Daniel-CA — APPROVED** [review]

**Comment by alfredoavanzosc** [comment]
Se cierra este PR, y se crea este nuevo https://github.com/avanzosc/sale-addons/pull/191

**Pull request closed without merging**